.ci/run_tests.sh

```diff
@@ -29,56 +43,70 @@ else
     export TEST_CMD="bin/busted $BUSTED_ARGS,postgres,db"
 fi
 
-if [[ "$KONG_TEST_COVERAGE" = true ]]; then
-    export TEST_CMD="$TEST_CMD --keep-going"
-fi
-
 if [ "$TEST_SUITE" == "integration" ]; then
+    integration_test_file=/tmp/integration_tests.$$.txt
+    trap "rm -f $integration_test_file" 0
+    # We're sorting the list of tests in a random order based on the
+    # pseudo random number generator of bash.  This should level the
+    # run time between the two splits better than performing a static
+    # split based on directory name.  Previously, all 05-proxy tests
+    # were run in the second split, which took three times as long as
+    # running all the other tests.
+    find spec/02-integration -name '*_spec.lua' \
+        | (RANDOM=0; while read line ; do echo "$RANDOM $line" ; done) \
+        | sort -n \
+        | awk '{print $2}' > $integration_test_file
```

Comment:
nit: you can use coreutils `shuf` for this

```suggestion
    find spec/02-integration -name '*_spec.lua' \
        | shuf \
        > "$integration_test_file"
```

Reply:
As far as I can tell, `shuf` does not provide a way to seed the PRNG.  It will randomize the sort order, which is not what we really want to do.  We want to shuffle, but we need the shuffle to always yield the same result for the same input set, hence `RANDOM=1` and `$RANDOM` as that always creates the same sequence of numbers.